test/allreduce_torchrun.py

```diff
@@ -53,4 +36,4 @@ def _mp_fn_xla_backend():
   if args.use_xla_backend:
     _mp_fn_xla_backend()
   else:
-    _mp_fn_xrt_init()
+    print('XRT has been deprecated, please set PJRT_DEVICE first')
```

Comment:
Please add the message in an `assert()` call.

Reply:
Thanks for pointing, `[test/allreduce_torchrun.py](https://github.com/pytorch/xla/pull/5222/files/5b94b8b4ec33bab9eac08a90cb26af659ffcc3e4#diff-9d538aef5bbef89bf7f0ced6629cfb4b98843d823fa2722451a959e98c37e820)` will be deleted in Will PR to delete XRT: https://github.com/pytorch/xla/pull/5240